variables.tf

```diff
@@ -89,13 +89,17 @@ variable "orca_default_bucket" {
   description = "Default ORCA S3 Glacier bucket to use if no overrides exist."
 }
 
+variable "orca_restore_retrieval_type" {
+  type        = string
+  description = "The Tier for the restore request. Valid values are 'Standard'|'Bulk'|'Expedited'."
+  default     = "Standard"
```

Comment:
Consider adding validation to this variable in all the modules so that users can only have one of the three. See https://www.terraform.io/language/values/variables#custom-validation-rules

Reply:
updated all modules variables.tf to 

```
variable "orca_default_restore_retrieval_type" {
  type        = string
  description = "The Tier for the restore request. Valid values are 'Standard'|'Bulk'|'Expedited'."
  validation {
    condition     = contains(["Standard", "Bulk", "Expedited"], var.orca_default_restore_retrieval_type)
    error_message = "Valid values are 'Standard'|'Bulk'|'Expedited'."
  }
}
```